ddtrace/span.py

```diff
@@ -373,7 +373,7 @@ def get_tag(self, key):
         """Return the given tag or None if it doesn't exist."""
         return self._meta.get(key, None)
 
-    def _get_tags(self):
+    def get_tags(self):
```

Comment:
Since these are public facing there should be docstrings (even though these methods are quite self-apparent)

Reply:
`get_tags` already has a docstring but will add one for `get_metrics` (and `get_metric` which was missing it).